massgov/mayflower-patternlab
# DP-8455: Allow empty search banner images to support setting with js

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

On mass.gov, the search banner URLs are set using javascript, pulling from a pool of random images.  The current implementation of this pattern is causing one extra HTTP request to the homepage for every page load due to the empty background-url CSS.

## Related Issue / Ticket

- [JIRA issue](https://jira.state.ma.us/browse/DP-8455)

## Steps to Test


1. On the Drupal side, pull in tag `5.17.0-alpha8455`.
2. Load the homepage and verify that no additional HTTP request is made to /.


## Screenshots
<img width="1226" alt="screen shot 2018-04-03 at 8 58 13 am" src="https://user-images.githubusercontent.com/654407/38250599-381c843e-371d-11e8-9c31-398318d778f3.png">


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
